src/nrf52840/README.md

```diff
@@ -60,6 +60,28 @@ After a successful build, the `elf` files can be found in `<path-to-ot-nrf528xx>
 $ arm-none-eabi-objcopy -O ihex build/bin/ot-cli-ftd ot-cli-ftd.hex
 ```
 
+### Alternative build using Docker
+
+Alternatively, it's possible to build using a Docker image.
+
+This can be useful in case of CI/CD builds or to build locally without installing the dependencies.
```

Comment:
What is CI/CD in this case? Do the abbreviations need to be opened?

Are there downsides? For example, is the user foregoing all the support options (USB CDC ACM support, Bootloader support, and so on)?

Reply:
I meant to say when building in a pipeline in the cloud, most of the time the build agent will run some generic OS and the build agent can be instructed to run our pipeline script in a specific container image.

In this case, building through the docker image can help better control the build environment over time and give better odds of a build not failing if the cloud provider updates the VM environment.

I don't know of any downsides but my tests were limited to the sample build I provided as an example in the PR description.

Reply:
Alright, that sounds good and doesn't necessitate any more clarification, I think.